src/playercards/ScriptedTarot.ttslua

```diff
@@ -10,30 +10,34 @@ READING = {
   "Chariot",
   "Wheel of Fortune"
 }
+doubleSize = false
 
 function onLoad()
   self.addContextMenuItem("Chaos", chaos)
   self.addContextMenuItem("Balance", balance)
   self.addContextMenuItem("Choice", choice)
   self.addContextMenuItem("Destiny (Campaign)", destiny)
   self.addContextMenuItem("Accept Your Fate", fate)
+  self.addContextMenuItem("Observed", observed)
+  self.addContextMenuItem("Damned", damned)
+  self.addContextMenuItem("Toggle Large Tarot", enlargeSize)
 end
 
-function chaos()
+function chaos(color)
   self.shuffle()
   self.takeObject({
-    position = self.getPosition() + CARD_OFFSET,
-    rotation = ORIENTATIONS[math.random(2)],
-    smooth = true
+    position = self.positionToWorld(BAG_OFFSET + CARD_OFFSET),
+    smooth = true,
+    callback_function = function(card) if math.random(2) == 2 then card.call("rotateSelfAndPreview", color) else card.setRotation(self.getRotation()) end end
```

Comment:
Looks like this is not getting the original bag rotation when it's rotated on its head.